examples/management_canister/src/caller/lib.rs

```diff
@@ -88,9 +88,10 @@ mod http_request {
         };
         let arg_raw = ic_cdk::export::candid::utils::encode_args((arg,))
             .expect("Failed to encode arguments.");
-        400_000_000u128 / 13
-            + 100_000u128 / 13
-                * (arg_raw.len() as u128 + "http_request".len() as u128 + max_response_bytes)
+        (3_000_000u128 + 60_000u128 * 13
```

Comment:
I have only found this place that needs to be changed because of the new proposed formula for calculating the cost of the http outcalls. 

Because it's a small change in the examples folder, I did not update  the changelog file as it does not seem necessary.